EvidenceApi.Tests/V1/E2ETests/EvidenceRequestsTest.cs

```diff
@@ -151,18 +187,177 @@ public async Task ReturnNotFoundWhenCannotFindEvidenceRequest()
         {
             var fakeId = "ed0f2bd2-df90-4f01-b7f1-d30e402386d0";
             var uri = new Uri($"api/v1/evidence_requests/{fakeId}", UriKind.Relative);
-            var responseFind = await Client.GetAsync(uri).ConfigureAwait(true);
+            var responseFind = await Client.GetAsync(uri);
             responseFind.StatusCode.Should().Be(404);
         }
 
+        [Test]
+        public async Task CreateDocumentSubmissionUnsuccessfulWhenNoDocumentTypeProvided()
```

Comment:
here are the tests from DocumentSubmissionTests -- you don't need to read through these.